src/helm/proxy/clients/huggingface_tokenizer.py

```diff
@@ -80,6 +80,8 @@ def load_tokenizer(hf_tokenizer_name: str, revision: Optional[str] = None):
                     hf_tokenizer_name = "Writer/palmyra-base"
                 elif tokenizer_name == "bigcode/starcoder":
                     hf_tokenizer_name = "bigcode/starcoder"
+                elif tokenizer_name == "tatsu-lab/alpaca-7b-wdiff":
```

Comment:
Does HELM handle the weight diff or is that a separate step?  It seems like we should just be using the standard LLaMA tokenizer here?

Reply:
Switched to the standard LLaMA tokenizer, which, believe it or not, is called `hf-internal-testing/llama-tokenizer`.

Reply:
thanks, I was just looking for a string that would give the LLaMA tokenizer. where did you find this string @yifanmai ?